src/routes/safe/components/Layout.jsx

```diff
@@ -104,6 +105,18 @@ const Layout = (props: Props) => {
 
   const { address, ethBalance, name } = safe
   const etherScanLink = getEtherScanLink('address', address)
+  const web3Instance = getWeb3()
+
+  const RenderAppTab = () => {
```

Comment:
@mikheevm I have tried to lazy load the tab, but seems not to be posible due to the continues re-renders of the app. what do you suggest?